### ISSUE TITLE
Satsense

### ISSUE THREAD

**Issue by bouweandela** [NLeSC/TEAM2018#24]
Title: tbd
Software: https://github.com/dynaslum/satsense
Authors: Berend, Bouwe, Elena, ..

Bouwe will write the paper. Software could be ready for publication by the end of 2018.

**Comment by romulogoncalves** [comment]
@bouweandela is this proposal still up for Team2018 November Sprint?

**Comment by bouweandela** [comment]
@romulogoncalves Yes

**Comment by romulogoncalves** [comment]
@bouweandela  is there project hours for this work? If yes, which one?

**Comment by bouweandela** [comment]
@romulogoncalves Yes, satsense is created for the Dynaslum project.

**Comment by bldevries-escience** [comment]
> 
> 
> @romulogoncalves Yes

I could work on this also... 2-3 days available. Let me know

**Comment by romulogoncalves** [comment]
@bouweandela were all the goals achieved? Could you summarize in 3 lines what was achieved? Can we consider it done and close this issue?

**Comment by bouweandela** [comment]
Unfortunately not all goals were achieved, since we did not get as much help as anticipated, so it was mostly just Berend and me working on our own project. Ben helped out one morning, but was unavailable the rest of the week.

Nevertheless, we made good progress:
- Documentation is now available on [readthedocs](https://satsense.readthedocs.io)
- Test coverage was improved to [a reasonable 80%](https://app.codacy.com/project/DynaSlum/satsense/dashboard#coverageData)
- All functionality was restored after a recent API change needed for memory/computational efficiency reasons, including in our most important [demonstration Jupyter Notebook](https://github.com/DynaSlum/satsense/blob/master/notebooks/demo_notebooks/Classification/Simple%20Example.ipynb)

I think this issue can only be closed once we have a draft of the paper submitted, right?

**Comment by romulogoncalves** [comment]
@bouweandela let me know once the paper is submitted.

**Comment by bouweandela** [comment]
Sure

**Comment by romulogoncalves** [comment]
@bouweandela what is the current status?
